src/Sdk/WebApi/WebApi/LaunchContracts.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace GitHub.Services.Launch.Contracts
+{
+    [DataContract]
+    public class ActionReferenceRequest
+    {
+        [DataMember(EmitDefaultValue = false, Name = "action")]
+        public string Action { get; set; }
+
+        [DataMember(EmitDefaultValue = false, Name = "version")]
+        public string Version { get; set; }
```

Comment:
this is currently called `Ref` instead of `Version`, what's the reason for the rename?

Reply:
This is copied pretty much directly from https://github.com/github/actions-dotnet/blob/179ed33568dd1ad80c457fb1c328c964e372accf/Launch/Sdk/Server/Models/GitHub.cs#L90-L152. Since the runner is now the one receiving the payload from launch, it needs to parse the JSON payload the same way. I don't believe this is a rename